core/src/main/resources/hudson/model/UsageStatistics/help-usageStatisticsCollected.jelly

```diff
@@ -38,24 +38,34 @@
             Each trial has a specific purpose and a defined end date, after which collection stops, independent of the installed versions of Jenkins or plugins.
             Once a trial is complete, the trial results may be aggregated and shared with the developer community.
         </p>
-        <p>
-            The following trials defined on this instance are active now or in the future:
```

Comment:
the help was ending with this as there's no trials currently (and an empty `dl`)